### ISSUE TITLE
Edit: Initiate section, opening paragraph

### ISSUE THREAD

**Issue by sharronrush** [w3c/wai-dynamic-planning#6]
The opening paragraph is clunky. It does not have to be that you are just "starting with accessibility"  in order to make these considerations and establish communication protocols. Suggest something like:

Accessibility goals are best achieved when they are fully integrated into overall project processes. To do this, align accessibility with existing organizational approaches; develop and articulate clear, measurable objectives; and include various stakeholders to secure understanding and broad support throughout the organization.

**Comment by davidberman** [comment]
Once the substantive decisions have been finalized as to what the introductory paragraph should say, I will be please to recommend the cleanup of editorial style I proposed in yesterday's survey.

**Comment by iadawn** [comment]
@sharronrush, I have incorporated part of your suggestion into an edited version of the introduction. How do you feel about 4ef04a3?

**Comment by sharronrush** [comment]
Yes it looks great, thanks Kevin!
